MetaMask/metamask-extension
# fix: adjust spacing of quote rate in swaps

## **Description**

The [PR to remove the legacy swaps code](https://github.com/MetaMask/metamask-extension/pull/27679) inadvertently caused a misalignment of the quote rate. This PR re-aligns it to be where it is expected.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28016?quickstart=1)

## **Manual testing steps**

1. Start a swap
2. Notice that the quote rate is back on one line and the value is left-aligned

## **Screenshots/Recordings**

### **Before**

![Screenshot 2024-10-11 at 11 32 09 AM](https://github.com/user-attachments/assets/aae5da2f-ae66-46f5-9168-6c6ed496a2a8)


### **After**

<img width="359" alt="Screenshot 2024-10-22 at 12 13 50 PM" src="https://github.com/user-attachments/assets/f5974e8a-62a6-464e-a037-b7ce38774f42">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
